applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -246,21 +247,15 @@ static int lc3_config_cb(struct bt_conn *conn, const struct bt_bap_ep *ep, enum
 		if (!audio_stream->conn) {
 			LOG_DBG("ASE Codec Config stream %p", (void *)audio_stream);
 
-			int ret;
-			uint32_t octets_per_sdu;
-
-			ret = le_audio_octets_per_frame_get(codec, &octets_per_sdu);
-			if (ret) {
-				LOG_ERR("Error retrieving octets frame:, %d", ret);
-				return ret;
+			ret = bitrate_check(codec);
+			if (!ret) {
+				LOG_WRN("Bitrate check failed");
```

Comment:
WRN or ERR?

Reply:
The connection will be rejected right after this, but that is expected behavior in case the bitrate or frequency is not supported, thus I think it should be a WRN. This goes for the comment below as well